store/helper/helper.go

```diff
@@ -314,7 +314,13 @@ func NewFrameItemFromRegionKey(key []byte) (frame *FrameItem, err error) {
 			var handle kv.Handle
 			_, handle, err = tablecodec.DecodeRecordKey(key)
 			if err == nil {
-				frame.RecordID = handle.IntValue()
+				if handle.IsInt() {
+					frame.RecordID = handle.IntValue()
+				} else {
+					str := handle.String()
+					frame.IndexName = "PRIMARY"
+					frame.IndexValues = strings.Split(str[1:len(str)-1], ", ")
```

Comment:
The `Split` here depends on the implementation of `handle.String()`. Should we add a method like `DatumStrings() []string` to the interface `kv.Handle`?

Reply:
Yeah, I guess there are some corner cases to use the `Split`.

Reply:
Or just add a function and pass handle as an argument.